test/scripts/e2e_client_runner.py

```diff
@@ -113,11 +113,22 @@ def _script_thread_inner(runset, scriptname):
     runset.running(scriptname, p)
     timeout = read_script_for_timeout(scriptname)
     try:
+        # Popen return code: https://docs.python.org/3/library/subprocess.html#subprocess.Popen.returncode
```

Comment:
excessive comment; retcode=0 meaning success is standard unix for decades. We can drop these two new lines of comment.

Reply:
code updated.